raspberry/bzzz/sensors/gnss.py

```diff
@@ -97,13 +99,14 @@ def __get_measurements_in_background_t(self, serial_path, baud):
                     lat_min = float(tokens[1]) % 100
                     latitude = deg_min_sec_to_decimal(lat_deg, lat_min, 
                                                       tokens[2])
+                    print(latitude)
 
                     lon_deg = int(float(tokens[3]) / 100)
                     lon_min = float(tokens[3]) % 100
                     longitude = deg_min_sec_to_decimal(lon_deg, lon_min, 
                                                        tokens[4])
-                elif msg_key == "$GPGSV" and len(tokens) > 5:
-                    altitude = float(tokens[5])
+                elif msg_key == "$GNGGA" and len(tokens) > 5:
```

Comment:
it was definitely working when I had written the code a couple of weeks ago. I can test again now

Reply:
Have you collected any data?

Reply:
yes this works. just tested with base station antenna

Reply:
![Figure_1](https://github.com/QUB-ASL/bzzz/assets/102968570/219f6e7a-2768-46f4-8b52-4103a0f7052b)

Reply:
this was out the window with no correction data

Reply:
It would be better to have actual data. You need to know the variance of the measurements both for your initialisation step and for the Kalman filter.